ldap/src/main/java/org/springframework/security/ldap/server/UnboundIdContainer.java

```diff
@@ -57,6 +57,12 @@ public UnboundIdContainer(String defaultPartitionSuffix, String ldif) {
 		this.ldif = ldif;
 	}
 
+	private void checkFilePath(String ldif) {
+		if (!StringUtils.hasText(ldif)) {
```

Comment:
Setting the LDIF is optional, so we shouldn't throw an exception, instead we should just avoid processing the LDIF.